apps/condo/bin/lib/constants.js

```diff
@@ -0,0 +1,19 @@
+const path = require('path')
+
+const EXEC_COMMAND = 'yarn workspace @app/condo node'
```

Comment:
inline all the consts. it's hard to read the code without this value. You don't need to reuse this value in anywhere alse